js/components/data-map.js

```diff
@@ -12,6 +17,10 @@
           var type = this.getAttribute('scale-type') || 'quantize';
           var scheme = this.getAttribute('color-scheme') || 'Blues';
           var steps = this.getAttribute('steps') || 5;
+          var units = this.getAttribute('units') || '';
```

Comment:
It is highly likely that units will be used in the data map, so I'm thinking I'll keep the units attribute for now. Fine by you @shawnbot ?